javascript/ditsmod/.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
It could be cool to have this file in the above folder (I mean in the javascript folder) so as we can have a common style on all js / ts / json files

And even a GitHub action that lint files directly

Reply:
Done!